spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/ResizeBilinear.scala

```diff
@@ -32,18 +32,74 @@ import scala.reflect.ClassTag
  * @tparam T Numeric type of parameter(e.g. weight, bias). Only support float/double now
  */
 class ResizeBilinear[T: ClassTag](val outputHeight: Int, val outputWidth: Int,
-  val alignCorners: Boolean)(implicit ev: TensorNumeric[T])
-  extends AbstractModule[Tensor[Float], Tensor[Float], T]{
+  val alignCorners: Boolean,
+  val dataFormat: DataFormat = DataFormat.NCHW)(implicit ev: TensorNumeric[T])
+  extends AbstractModule[Tensor[T], Tensor[T], T]{
 
   private val ys = (1 to (outputHeight + 1)).map(i => InterpolationWeight(0, 0, 0)).toArray
   private val xs = (1 to (outputWidth + 1)).map(i => InterpolationWeight(0, 0, 0)).toArray
 
   import ResizeBilinear._
 
-  override def updateOutput(input: Tensor[Float]): Tensor[Float] = {
+  override def updateOutput(input: Tensor[T]): Tensor[T] = {
```

Comment:
why change to T? Is there anywhere use double image input?

Reply:
Hi, @yiheng . I don't know the exact case where double image is used as input, but I thought it is make sense to support double, as other layers dealing with image like SpatialConvolution, SpatialMaxPooling, .etc all supports double.

Reply:
I think maybe no people want to use double tensor as image input. So we need not support this. If there's such require in the future, we can add then.